samples/usb/usb_uart_bridge/README.rst

```diff
@@ -13,11 +13,13 @@ The USB-UART bridge acts as a serial adapter, exposing 2 UART pairs to a USB hos
 Requirements
 ************
 
-* One of the following development boards:
+The sample supports the following nRF9160-based device:
 
-  * |Thingy91|
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set10_start
```

Comment:
If I understand this reference correctly, it should be changed to reflect `thingy91_nrf52840` as build target